### PR TITLE
feat: add seventh day guests

### PR DESCRIPTION
**Pull request by LayZeeDK**

### CODE REVIEW

File: src/assets/content/angular-calendar.json [hunk 1]
```diff
@@ -78,7 +78,16 @@
       }
     },
     {
-      "day": 7
+      "day": 7,
+      "content": {
```
Comment (9kubczas4) [inline src/assets/content/angular-calendar.json:82]: Hi @LayZeeDK,
The schema was slight changed. Now we can display more than one guest per day, so it should be:
```json
{
      "day": 7,
      "contents": [
        {
          "author": {...},
          "videoSrc": "PLACEHOLDER"
        }
      ]
    },
```.

Thank you :)

**Comment by LayZeeDK** [comment]
Waiting for the second day 7 guest 😃

File: src/assets/content/angular-calendar.json [hunk 2]
```diff
@@ -90,7 +90,18 @@
       ]
     },
     {
-      "day": 7
+      "day": 7,
+      "contents": [
```
[the remark this answers is not in the document]
Comment (crutchcorn) [reply src/assets/content/angular-calendar.json:94]: Added!

File: src/assets/content/angular-calendar.json [hunk 3]
```diff
@@ -90,7 +90,27 @@
...
+          "videoSrc": "https://streamable.com/e/zc1tlp"
+        },
+        {
+          "author": {
+            "name": "Corbin Crutchley",
+            "avatarUrl": "https://avatars.githubusercontent.com/u/9100169?v=4",
+            "brandImageUrl": "https://avatars.githubusercontent.com/u/53884590?v=4",
+            "brandExternalUrl": "https://framework.guide"
+          },
+          "videoSrc": "https://streamable.com/e/ii63t7"
```
Comment (LayZeeDK) [inline src/assets/content/angular-calendar.json:111]: issue: We might want to boost the volume a notch

File: src/assets/content/angular-calendar.json [hunk 4]
```diff
@@ -90,7 +90,27 @@
       ]
     },
     {
-      "day": 7
+      "day": 7,
+      "contents": [
+        {
+          "author": {
+            "name": "Lars Gyrup Brink Nielsen",
```
[the remark this answers is not in the document]
Comment (LayZeeDK) [reply src/assets/content/angular-calendar.json:97]: Okay, then this PR is good to go.

**Pull request merged**